boards/c030-u201/Makefile.dep

```diff
@@ -0,0 +1,10 @@
+USEMODULE += stm32_periph_uart_hw_fc
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+  # USEMODULE += si7034 # TODO: add when si70xx driver is updated
```

Comment:
What is missing in this driver ?

Reply:
I'm not sure to remember, but there are 2 sub-families and `si7034` is not in the one which was imlpemented.